manifests/cf-manifest/deployments/050-logsearch.yml

```diff
@@ -8,106 +8,190 @@ releases:
   version: 4
   sha1: c92ce357c5e1e77a4fb7944d58a1971c7aee06b0
 
-disk_pools:
-- name: elasticsearch_master
-  disk_size: 102400
-  cloud_properties:
-    type: gp2
-
-- name: elasticsearch_data
-  disk_size: 102400
-  cloud_properties:
-    type: gp2
-
-- name: queue
-  disk_size: 102400
-  cloud_properties:
-    type: gp2
-
-- name: cluster_monitor
-  disk_size: 102400
-  cloud_properties:
-    type: gp2
-
 resource_pools:
-- name: elasticsearch_master
-  network: cf1
+- name: elastic_master_z1
+  network: (( grab resource_pools.medium_z1.network ))
   stemcell: (( grab meta.stemcell ))
   env: (( grab meta.default_env ))
   cloud_properties:
-    availability_zone: eu-west-1a
-    instance_type: t2.micro
+    availability_zone: (( grab resource_pools.medium_z1.cloud_properties.availability_zone ))
+    instance_type: (( grab resource_pools.medium_z1.cloud_properties.instance_type ))
+    elbs:
+      - (( grab terraform_outputs.elastic_master_elb_name ))
 
-- name: elasticsearch_data
-  network: cf1
+- name: elastic_master_z2
```

Comment:
From meatspace: we've agreed to combine them for now on the basis that:

- we don't want to significantly increase the deploy times for dev environments right now
- we want parity with dev and production environments
- we don't expect *that* much data
- however we do need to turn off debug logging, in a separate story